### PR TITLE
feat: export `CommonResolutions` constant for helping configure new compositions

### PR DESCRIPTION
**Pull request by alittlebitweird**
i.e.

```
import { Editframe, CommonResolutions } from '@editframe/editframjs'
...
const composition = editframe.videos.new({dimensions: CommonResolutions._1080p})
```

thoughts?

### CODE REVIEW

**Comment by brycedev** [comment]
i need more than a moment to think on this

File: src/constant/resolutions/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Size } from 'constant/video/layers'
+
+export const CommonResolution: Record<string, Size> = {
+  _1080p: { height: 1080, width: 1920 },
+  _1080pVertical: { height: 1920, width: 1080 },
+  _1080x1080: { height: 1080, width: 1080 },
+  _4k: { height: 2160, width: 3840 },
```
Comment (brycedev) [inline src/constant/resolutions/index.ts:7]: let's get rid of 4k and 8k

Comment (brycedev) [inline src/constant/resolutions/index.ts:7]: let's get rid of 4k and 8k

**Review by brycedev — CHANGES_REQUESTED** [review]

**Comment by brycedev** [comment]
balking a little at these names, but this is certainly a good step for now 👍

**Comment by brycedev** [comment]
@alittlebitweird bumping this. are we okay to get this merged in?

**Comment by alittlebitweird** [comment]
> Reference

Yeah I've also just been thinking of whether there's a nicer way to do this but I think its fine for now

**Pull request merged**